cellrank/tl/estimators/_base_estimator.py

```diff
@@ -555,6 +532,60 @@ def compute_absorption_probabilities(
 
         self._write_absorption_probabilities(time=start, extra_msg=extra_msg)
 
+    def compute_lineage_priming(
+        self, method: str = "kl_divergence", early_cells: Optional[Sequence[str]] = None
+    ) -> pd.Series:
+        """
+        TODO.
+
+        Parameters
+        ----------
+        method
+            TODO. Valid options are:
```

Comment:
Suggestion:

"The method used to compute the degree of lineage priming. Valid options are: 
- `kl_divergence`: as in [cite Stemnet], computes the kl divergence between the fate probabilities of a cell and the average fate probabilities. Computation of average fate probabilities can be restricted to a set of user-defined early cells, see [cite parameter]. The further from average a cell's fate probabilities, the more committed it is. 
- `entropy`, as in [palantir], computes entropy over a cell's fate probabilities. The lower a cell's entropy over fate probabilities, the more committed it is. "